### DISCUSSION TITLE
1.0 possible release date

### DISCUSSION THREAD

**Discussion by paladinperry** [rg35xx-cfw/Koriki#66] [General]
Hi guys, I've been sticking with koriki as its the best looking UI with the best features
I am desperately wanting the 1.0 release.
any news on a possible launch period ?

**Comment by acmeplus** [comment]
No date but there's an interim update:

https://github.com/rg35xx-cfw/Koriki/releases/tag/koriki_rg35xx_v0.9.2-test

**Reply by paladinperry** [reply]
Thank you so much for this ! 
Was getting worried this project was abandoned, I can't deal with that lol.
This is my favourite and sole OS for RG35XX
